Lib/test/test_enum.py

```diff
@@ -2950,9 +2950,7 @@ class Color(StrMixin, AllMixin, IntFlag):
         self.assertEqual(Color.ALL.value, 7)
         self.assertEqual(str(Color.BLUE), 'blue')
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
-    @unittest.skipIf(sys.platform == "win32", "TODO: RUSTPYTHON, universal newlines")
+    @unittest.skip("TODO: RUSTPYTHON, nondeterministic on win32 (universal newlines) fails on other platforms")
```

Comment:
The test you're looking for is actually in a completely different class (around line 2520).

Reply:
Ah, that's confusing, that would explain why nothing I was doing worked. Thanks :sweat_smile: